assignment2/q1.py

```diff
@@ -0,0 +1,36 @@
+# Print AncestorsGiven a Binary Tree and a key, 
+# write a function that prints all the ancestors of the key in the givenbinary tree.
+class Node:
+ 
+    
+    def __init__(self, data):
+        self.data = data 
+        self.left = None
+        self.right = None
+
+    def printAncestors(root, target):
+
+        if root == None:
+            return False
+
+        if root.data == target:
+            return True
+
+
+        if (printAncestors(root.left, target) or printAncestors(root.right, target)):
+            print root.data,
```

Comment:
sorry,
When I commited the files I forgot to change some lines.
I will change the solution